ada/views.py

```diff
@@ -0,0 +1,33 @@
+from django.http import HttpRequest, HttpResponse
+from django.shortcuts import HttpResponseRedirect, render
+
+from .models import RedirectUrl, RedirectUrlsEntry
+
+
+def clickActivity(request: HttpRequest, pk: int, whatsappid: int) -> HttpResponse:
+    try:
+        redirect_url = RedirectUrl.objects.get(id=pk)
+    except (ValueError, RedirectUrl.DoesNotExist):
+        return render(request, "index.html", {"error": "HTTP Bad Request"})
+    store_url_entry = RedirectUrlsEntry(url=redirect_url)
+    store_url_entry.save()
+    destination_url = f"{redirect_url.symptom_check_url}" f"/?whatsappid={whatsappid}"
+    return HttpResponseRedirect(destination_url)
+
+
+def default_page(request: HttpRequest, pk: int) -> HttpResponse:
+    try:
+        whatsappid = request.GET.get("whatsappid")
```

Comment:
In `urls.py`, we're doing a `confirmredirect/<int:pk>/<int:whatsappid>`, so the whatsapp ID is in the URL, not in the querystring.